sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormField.cs

```diff
@@ -19,19 +19,15 @@ internal FormField(string name, int pageNumber, KeyValuePair_internal field, IRe
             Name = name;
 
             BoundingBox labelBoundingBox = field.Key.BoundingBox == null ? default : new BoundingBox(field.Key.BoundingBox);
-            IReadOnlyList<FormContent> labelFormContent = default;
-            if (field.Key.Elements != null)
-            {
-                labelFormContent = ConvertTextReferences(field.Key.Elements, readResults);
-            }
+            IReadOnlyList<FormContent> labelFormContent = field.Key.Elements != null
+                ? ConvertTextReferences(field.Key.Elements, readResults)
+                : new List<FormContent>();
```

Comment:
It was decided that `TextContent` will be an empty list when there's no text element. We're doing this in other places as well (like in `FormTableCell`).